### PR TITLE
Resprites ID cards & chip

### PR DESCRIPTION
**Pull request by Twaticus**
## About The Pull Request

Resprites tier ID cards + chips.
Resprites extra ID cards(prison, black, centcom, flame, rainbow, carp - unsure which are used at this point but theyre consistent now)
Removes on-mob ID cards and PDA in ID slot sprites. Too much visual clutter, unnecessary.
image
tier0, tier1, tier2, tier3, tier4, tier5 - w/ chip, no access
same tiers - w/ chip, 1 access
same tiers - w/chip, 2 access

this is mostly for sprite framework, i plan to revisit this again.

## Changelog
:cl:
imageadd: Resprites ID cards & chips
imagedel: Removes on-mob ID sprites
/:cl:

### CODE REVIEW

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -435,46 +434,46 @@
 	name = "[name_string] ([assignment])"
 
 /obj/item/card/id/tier0
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:437]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 2]
```diff
@@ -435,46 +434,46 @@
...
 	icon_state = "tier0"
 	access_tier = 0
 
 /obj/item/card/id/tier1
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:442]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 3]
```diff
@@ -435,46 +434,46 @@
...
 	icon_state = "tier1"
 	chip_slots = 2
 	access_tier = 1
 
 /obj/item/card/id/tier2
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:448]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 4]
```diff
@@ -435,46 +434,46 @@
...
 	icon_state = "tier2"
 	chip_slots = 2
 	access_tier = 2
 
 /obj/item/card/id/tier3
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:454]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 5]
```diff
@@ -435,46 +434,46 @@
...
 	icon_state = "tier3"
 	chip_slots = 2
 	access_tier = 3
 
 /obj/item/card/id/tier4
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:460]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 6]
```diff
@@ -435,46 +434,46 @@
...
 	icon_state = "tier4"
 	chip_slots = 2
 	access_tier = 4
 
 /obj/item/card/id/tier5
 	name = "gold identification card"
 	desc = "A golden card which shows power and might."
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:468]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 7]
```diff
@@ -833,7 +825,7 @@
 /obj/item/card/id/chip_programmer
 	name = "additional access chip programmer"
 	desc = "A card used to program new access chips."
-	icon = 'icons/obj/new_id.dmi'
+	icon = 'icons/obj/card.dmi'
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:828]:
```suggestion
```

File: code/game/objects/items/cards_ids.dm [hunk 8]
```diff
@@ -833,7 +825,7 @@
 /obj/item/card/id/chip_programmer
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/cards_ids.dm:825]: it would be nice if this one had it's own icon.

**Comment by Sparkezel** [comment]
Could you please change how the yellow part on ID's looks? instead of like copper pins maybe try to make it look like actual chips could be put in

File: code/game/objects/items/cards_ids.dm (hunk 8 above)
Comment (Twaticus) [reply code/game/objects/items/cards_ids.dm:825]: i dont even know what that is

**Comment by Twaticus** [comment]
STFU ILL GET TO IT BOT

File: code/game/objects/items/cards_ids.dm (hunk 8 above)
Comment (Twaticus) [reply code/game/objects/items/cards_ids.dm:819]: gonna tackle that in the next batch, i have some ideas

**Pull request merged**